BE/PoroDev.Common/Models/UserModels/Data/DataUserModel.cs

```diff
@@ -31,7 +31,7 @@ public class DataUserModel
         [JsonIgnore(Condition = JsonIgnoreCondition.Always)]
         public DateTime DateCreated { get; set; }
 
-        [JsonIgnore(Condition = JsonIgnoreCondition.Always)]
+        //[JsonIgnore(Condition = JsonIgnoreCondition.Always)]
```

Comment:
Should this be removed?

Reply:
Yeah was changed for testing forgot to uncomment